simplesat/repository.py

```diff
@@ -52,12 +52,13 @@ def __len__(self):
         )
 
     def __contains__(self, package_metadata):
-        return (
-            package_metadata in
-            self._name_to_packages.get(
-                package_metadata.name, self._default_factory()
+        if package_metadata.name in self._name_to_packages:
```

Comment:
Since we're not adding anything here, it really doesn't matter what the default factory is.

```python
candidates = self._name_to_packages.get(package_metadata.name, ())
return package_matadata in candidates
```
seems nicer to me.

Too late now. Don't change it.